hello.qmd

```diff
@@ -2,10 +2,44 @@
 
 ## What is Quarto?
 
+Quarto is an open-source scientific and technical publishing system built on Pandoc.
+It provides a unified authoring framework for data science, combining code, results, and narrative and outputs to HTML, PDF, Word, and more in a seamless manner.
+A few of the key features of Quarto that are applicable to Quarto users at any level are listed below:
+
+1.  **Markdown authoring**: Quarto documents are authored using **markdown**, a straightforward plain text format that allows you to add formatting elements to plain text documents using a simple syntax.
+
+2.  **Code integration**: Quarto extends the code capabilities of Pandoc by embedding code and output from various languages, including Python, R, Julia, and JavaScript.
+    This is made possible by execution engines like Jupyter, knitr, and Observable.
+
+3.  **Technical writing extensions**: Quarto includes extensions to Pandoc markdown specifically designed for technical writing that cover features like cross-references, sub-figures, layout panels, hoverable citations and footnotes, and more.
+
+4.  **Project system**: Quarto provides a project system for managing groups of documents with shared options to produce aggregate output such as manuscripts, websites, and books.
+
+5.  **Editor compatibility:** Quarto supports authoring in various editors and notebooks, including JupyterLab, RStudio, and VS Code.
+    Many of these editors generally offer a visual markdown editor that is specifically designed to work well with Quarto, providing a productive writing interface for composing long-form documents.
+
+The best place to learn about Quarto and explore its capabilities, aside from this book, is [quarto.org](https://quarto.org/).
+
+Quarto is available under the [MIT license](https://quarto.org/license.html).
+
+Sometimes when we say "Quarto" we mean "a Quarto document", i.e., a plain-text file with the extension `.qmd`, a Quarto markdown file.
```

Comment:
The many clauses here bother me a little. This was my best attempt to reduce them:
```suggestion
Sometimes when we say "Quarto" we mean "a Quarto document", i.e., a plain-text Quarto markdown file with the extension `.qmd`.
```